addons/project/models/project.py

```diff
@@ -410,17 +407,10 @@ def action_view_account_analytic_line(self):
         return action
 
     def action_view_all_rating(self):
-        """ return the action to see all the rating of the project, and activate default filters """
-        if self.portal_show_rating:
-            return {
-                'type': 'ir.actions.act_url',
-                'name': "Redirect to the Website Projcet Rating Page",
-                'target': 'self',
-                'url': "/project/rating/%s" % (self.id,)
-            }
+        """ return the action to see all the rating of the project and activate default filters"""
         action = self.env['ir.actions.act_window'].for_xml_id('project', 'rating_rating_action_view_project_rating')
         action['name'] = _('Ratings of %s') % (self.name,)
-        action_context = ast.literal_eval(action['context']) if action['context'] else {}
+        action_context = safe_eval(action['context']) if action['context'] else {}
```

Comment:
Why do you need to convert the ``literal_eval`` into a ``safe_eval`` ?

Reply:
I don't remember, I think it was just some test (this code went back and forth a few times during my dev) and the original code works so I changed it back as it was.